### PR TITLE
user/fonts-inter: new package

### PR DESCRIPTION
**Pull request by VyacheslavKostromin**

### CODE REVIEW

File: user/fonts-inter/template.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+maintainer = "beb <beb_@tutamail.com>"
+license = "OFL-1.1"
+url = "https://rsms.me/inter"
+source = f"https://github.com/rsms/inter/releases/download/v{pkgver}/Inter-{pkgver}.zip"
+sha256 = "ff970a5d4561a04f102a7cb781adbd6ac4e9b6c460914c7a101f15acb7f7d1a4"
+
+
+def do_install(self):
+    for f in (self.cwd).glob("*.ttf"):
+        self.install_file(f, "usr/share/fonts/inter")
```
Comment (nekopsykose) [inline user/fonts-inter/template.py:14]:
```suggestion
    self.install_file("./*.ttf", "usr/share/fonts/inter")
```

identical

File: user/fonts-inter/template.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    self.install_file(
+        "extras/ttf/*.ttf", "usr/share/fonts/inter", glob=True
+    )
+
+    self.install_file(
+        "extras/otf/*.otf", "usr/share/fonts/inter", glob=True
+    )
+
+    self.install_file("LICENSE.txt", "usr/share/fonts/inter")
```
Comment (nekopsykose) [inline user/fonts-inter/template.py:23]:
```suggestion
    self.install_license("LICENSE.txt")
```

File: user/fonts-inter/template.py (hunk 1 above)
Comment (nekopsykose) [reply user/fonts-inter/template.py]: note the glob that iadded in an edit

**Comment by nekopsykose** [comment]
when fonts-liberation was mentioned, you should look at it and see that -ttf/-otf are separate

**Comment by VyacheslavKostromin** [comment]
Should I make the variable-ttf variant separate too?

**Comment by nekopsykose** [comment]
i think not, just two packages probably..

**Comment by VyacheslavKostromin** [comment]
is that right?

**Comment by nekopsykose** [comment]
sure

**Comment by VyacheslavKostromin** [comment]
i forgot to delete the comment

**Comment by VyacheslavKostromin** [comment]
I just made a terrible mistake, right?

File: user/fonts-inter/template.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+def do_install(self):
+    self.install_file("*.ttf", "usr/share/fonts/inter", glob=True)
+
+    self.install_file("extras/ttf/*.ttf", "usr/share/fonts/inter", glob=True)
+
+    self.install_file("extras/otf/*.otf", "usr/share/fonts/inter", glob=True)
+
+def post_install(self):
+   self.install_license("LICENSE.txt")
```
Comment (nekopsykose) [inline user/fonts-inter/template.py:20]:
```suggestion
    self.install_file("extras/ttf/*.ttf", "usr/share/fonts/inter", glob=True)
    self.install_file("extras/otf/*.otf", "usr/share/fonts/inter", glob=True)
    self.install_license("LICENSE.txt")
```

**Comment by nekopsykose** [comment]
you should look at how to fixup/squash commits, the using_git.md gives an example

**Comment by VyacheslavKostromin** [comment]
I hope everything is fine now?

**Comment by nekopsykose** [comment]
the commits aren't squashed, it's now 5 commits

you'll have to learn how to use `git rebase -i` :)

**Comment by triallax** [comment]
template needs to be formatted with black/ruff

File: user/fonts-inter/template.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+url = "https://rsms.me/inter"
+source = f"https://github.com/rsms/inter/releases/download/v{pkgver}/Inter-{pkgver}.zip"
+sha256 = "ff970a5d4561a04f102a7cb781adbd6ac4e9b6c460914c7a101f15acb7f7d1a4"
+options = ["empty"]
+
+def do_install(self):
+    self.install_file("*.ttf", "usr/share/fonts/inter", glob=True)
+    self.install_file("extras/ttf/*.ttf", "usr/share/fonts/inter", glob=True)
+    self.install_file("extras/otf/*.otf", "usr/share/fonts/inter", glob=True)
+    self.install_license("LICENSE.txt")
```
Comment (triallax) [inline user/fonts-inter/template.py:16]: does OFL-1.1 need to be installed?

Comment (triallax) [reply user/fonts-inter/template.py:16]: it seems like the answer might be yes:

> 2) Original or Modified Versions of the Font Software may be bundled,
> redistributed and/or sold with any software, provided that each copy
> contains the above copyright notice and this license. These can be
> included either as stand-alone text files, human-readable headers or
> in the appropriate machine-readable metadata fields within text or
> binary files as long as those fields can be easily viewed by the user.

if so, we should lint this

Comment (triallax) [reply user/fonts-inter/template.py:16]: but anyway you don't need to do anything for this pr in particular, installing the license is correct here

**Pull request merged**